scripts/setup_osx.sh

```diff
@@ -1,38 +1,267 @@
+GARAGE_DIR="$(dirname $(script_dir_path))"
+cd "${GARAGE_DIR}"
+
+# File where environment variables are stored
+BASH_PROF="${HOME}/.bash_profile"
+
+# Install dependencies
+echo "Installing garage dependencies"
+echo "You will probably be asked for your sudo password"
+
+# Homebrew is required first to install the other dependencies
+hash brew 2>/dev/null || {
+  # Install the Xcode Command Line Tools
+  xcode-select --install
+  # Install Homebrew
+  /usr/bin/ruby -e "$(curl -fsSL \
+    https://raw.githubusercontent.com/Homebrew/install/master/install)"
+}
+
+# For installing garage: bzip2, git, glfw, unzip, wget
+# For building glfw: cmake
+# Required for OpenAI gym: cmake boost boost-python ffmpeg sdl2 swig wget
+# Required for OpenAI baselines: cmake openmpi
+brew update
+brew install \
+  bzip2 \
+  git \
+  glfw \
+  unzip \
+  wget \
+  cmake \
+  boost \
+  boost-python \
+  ffmpeg \
+  sdl2 \
+  swig \
+  openmpi
+# Required by MuJoCo
+brew reinstall gcc@7 --without-multilib
+
+# Leave a note in ~/.bash_profile for the added environment variables
+if [[ "${_arg_modify_bashrc}" = on ]]; then
+  echo -e "\n# Added by the garage installer" >> "${BASH_PROF}"
+fi
+
+# Set up MuJoCo
+if [[ ! -d "${HOME}/.mujoco/mjpro150" ]]; then
+  mkdir "${HOME}"/.mujoco
+  MUJOCO_ZIP="$(mktemp -d)/mujoco.zip"
+  wget https://www.roboti.us/download/mjpro150_osx.zip -O "${MUJOCO_ZIP}"
+  unzip -u "${MUJOCO_ZIP}" -d "${HOME}"/.mujoco
+else
+  print_warning "MuJoCo is already installed"
+fi
+# Configure MuJoCo as a shared library
+export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:${HOME}/.mujoco/mjpro150/bin"
+LD_LIB_ENV_VAR="LD_LIBRARY_PATH=\"\$LD_LIBRARY_PATH:${HOME}/.mujoco/mjpro150"
+LD_LIB_ENV_VAR="${LD_LIB_ENV_VAR}/bin\""
+if [[ "${_arg_modify_bashrc}" = on ]]; then
+  echo "export ${LD_LIB_ENV_VAR}" >> "${BASH_PROF}"
+fi
+
+# Set up conda
+if [[ ! -d "${HOME}/miniconda2" ]]; then
+  CONDA_INSTALLER="$(mktemp -d)/miniconda.sh"
+  wget https://repo.continuum.io/miniconda/Miniconda2-latest-MacOSX-x86_64.sh \
+    -O "${CONDA_INSTALLER}"
+  chmod u+x "${CONDA_INSTALLER}"
+  bash "${CONDA_INSTALLER}" -b -u
+  # Add conda to executable programs
+  CONDA_SH="${HOME}/miniconda2/etc/profile.d/conda.sh"
+  if [[ "${_arg_modify_bashrc}" = on ]]; then
+    echo ". ${CONDA_SH}" >> "${BASH_PROF}"
+  fi
+fi
+# Export conda in this script
+. "${CONDA_SH}"
+conda update -q -y conda
+
+# Create conda environment
 conda env create -f environment.yml
-conda env update
+if [[ "${?}" -ne 0 ]]; then
+  print_error "Error: conda environment could not be created"
+fi
+
+# Extras
+conda activate garage
+# Prevent pip from complaining about available upgrades
+pip install --upgrade pip
+# Fix Box2D install
+# See https://github.com/openai/gym/issues/100
+# See https://github.com/pybox2d/pybox2d/issues/82
+pip uninstall -y Box2D Box2D-kengz box2d-py
+PYBOX2D_DIR="$(mktemp -d)/pybox2d"
+git clone https://github.com/pybox2d/pybox2d "${PYBOX2D_DIR}"
```

Comment:
Did `pip install Box2D-kengz --no-binary Box2D-kengz` not work on your system?

Checking out and installing from source should be considered a last resort. Your checkout is using `master`, not a release tag or hash, so it could break without warning for no reason.

Reply:
I didn't try installing Box2D kengz, sorry for not noticing this one before. I will try it.

Reply:
It worked!